src/Conversion/ONNXToMhlo/Math/Elementwise.cpp

```diff
@@ -12,9 +12,9 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "mlir-hlo/utils/broadcast_utils.h"
 #include "src/Conversion/ONNXToMhlo/ONNXToMhloCommon.hpp"
 #include "src/Dialect/ONNX/ShapeInference/ONNXShapeHelper.hpp"
+#include "stablehlo/dialect/BroadcastUtils.h"
```

Comment:
I think there's a bit of cleanup that can happen in the `CMakeLists.txt` file as well.